packages/@ember/object/lib/computed/reduce_computed_macros.ts

```diff
@@ -429,7 +429,8 @@ export function mapBy(dependentKey: string, propertyKey: string) {
 }
 
 /**
-  Filters the array by the callback.
+  Filters the array down to the elements that pass the test implemented by the
+  callback.
```

Comment:
If we are tweaking this, I think the best way to do so is probably to refer to `Array.prototype.filter`.

```suggestion
  Filters the array by the callback, like the `Array.prototype.filter` method.
```